pkg/tests/tasks/security/certificate/cert_manager_test.go

```diff
@@ -0,0 +1,82 @@
+package certificate
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/maistra/maistra-test-tool/pkg/app"
+	"github.com/maistra/maistra-test-tool/pkg/util/check/assert"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/pod"
+	"github.com/maistra/maistra-test-tool/pkg/util/retry"
+	"github.com/maistra/maistra-test-tool/pkg/util/shell"
+	"github.com/maistra/maistra-test-tool/pkg/util/test"
+)
+
+func TestCertManager(t *testing.T) {
+	test.NewTest(t).Id("T38").Groups(test.Full, test.ARM, test.InterOp).Run(func(t test.TestHelper) {
+
+		ns := "bookinfo"
+		cert := "cert-manager"
+		t.Cleanup(func() {
+			oc.DeleteNamespace(t, cert)
+			oc.RecreateNamespace(t, ns)
+		})
+
+		t.LogStep("uninstall the SMCP")
+		oc.RecreateNamespace(t, meshNamespace)
+
+		t.LogStep("Install cert-manager")
+		shell.Execute(t,
+			fmt.Sprintf(`helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --version v1.11.0 --set installCRDs=true`),
```

Comment:
This fails for me: 
```    
shell.go:137: Running command: helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --version v1.11.0 --set installCRDs=true
    shell.go:143: Command output: 
        Error: INSTALLATION FAILED: repo jetstack not found
```

The test should probably add the repo as the first step.

Reply:
added the repo step